.circleci/config.yml

```diff
@@ -47,17 +49,29 @@ aliases:
         echo "ruby-${_RUBY_VERSION}" > ~/.ruby-version
         echo 'chruby ruby-${_RUBY_VERSION}' >> ~/.bash_profile
 
-version: 2
+  - &bundle_install
+    run:
+      name: bundle install
+      command: |
+        gem install bundler -v $(cat Gemfile.lock | tail -1 | tr -d " ")
+        bundle config set --local path .bundle
```

Comment:
This is to fix this deprecation warning.

> [DEPRECATED] The `--path` flag is deprecated because it relies on being remembered across bundler invocations, which bundler will no longer do in future versions. Instead please use `bundle config set --local path '.bundle'`, and stop using this flag